host/libubertooth/src/ubertooth.c

```diff
@@ -447,12 +443,21 @@ void rx_file(FILE* fp, btbb_piconet* pn)
 	int r = btbb_init(max_ac_errors);
 	if (r < 0)
 		return;
-	stream_rx_file(fp, cb_br_rx, pn);
+
+	ubertooth_t* ut = ubertooth_init();
+	if (ut == NULL)
+		return;
+
+	stream_rx_file(ut, fp, cb_br_rx, pn);
 }
 
 void rx_btle_file(FILE* fp)
 {
-	stream_rx_file(fp, cb_btle, NULL);
+	ubertooth_t* ut = ubertooth_init();
+	if (ut == NULL)
+		return;
+
+	stream_rx_file(ut, fp, cb_btle, NULL);
```

Comment:
Ignore this, I was confusing the library initialisation with something actually looking for an Ubertooth.